### ISSUE TITLE
Specify port

### ISSUE THREAD

**Issue by Vliro** [sgeisler/cargo-remote#15]
I am trying to use the program however I do not know how to specify the port used for communication. It seems to always be 22 but the port for connection is 23. How do I change it?

**Comment by sgeisler** [comment]
Hi, there is an [open PR](https://github.com/sgeisler/cargo-remote/pull/13) for this which unfortunately has a few problems so I didn't merge it. It seems to work though, so you can use it at your own risk if you want. You could also fork it and address the problems to get it merged.

**Comment by Vliro** [comment]
Thanks @sgeisler , that worked great.
However, I notice that the copy back seems to copy all assemblies as well, if I am running debug that is. Is there an ability to only copy the relevant binaries?

**Comment by sgeisler** [comment]
I don't think so. When I developed it I had a slow CPU, but a 1GBit network to my build server so I didn't optimize for transfer size.

cargo-remote would need to understand cargo files much better for that to work (needs to find all binary names). PRs are always welcome.
